src/main/java/com/example/eventapplication/user/service/UserServiceImpl.java

```diff
@@ -28,15 +30,18 @@ public void createUser(UserDto userDto) {
 
     @Override
     public List<NotificationDto> getAllNotificationsForUser(UUID technicalUserId) {
-        return userRepository.findUserByTechnicalUserId(technicalUserId).get().getNotificationEntityListForUser()
+        Optional<UserEntity> userEntity = userRepository.findUserByTechnicalUserId(technicalUserId);
+        userEntity.orElseThrow(() -> new UserServiceException("Notifications cannot be displayed " +
+                "because the specified user does not exist!"));
+        return userEntity.get().getNotificationEntityListForUser()
```

Comment:
```
return userRepository.findUserByTechnicalUserId(technicalUserId)
.map(e->e. getNotificationEntityListForUser())
.orElseThrow(() -> new UserServiceException("Notifications cannot be displayed " +
                 "because the specified user does not exist!"))
  
```